homeassistant/components/sonos/media_player.py

```diff
@@ -696,6 +702,11 @@ def remove_from_queue(self, queue_position: int = 0) -> None:
         """Remove item from the queue."""
         self.coordinator.soco.remove_from_queue(queue_position)
 
+    @soco_error()
+    def party_mode_services(self) -> None:
```

Comment:
Please rename to partymode so that the function name is consistent with the service name.